pkg/resource/replication_group/post_set_output.go

```diff
@@ -0,0 +1,70 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package replication_group
+
+import (
+	"context"
+
+	svcapitypes "github.com/aws-controllers-k8s/elasticache-controller/apis/v1alpha1"
+	"github.com/aws/aws-sdk-go/service/elasticache"
```

Comment:
recommend aliasing this package as `svcsdk` as we do elsewhere in all the `pkg/resource` templates:

https://github.com/aws-controllers-k8s/code-generator/blob/cf8f94fd9835b500cbf736fcea48efe0ca37f8a1/templates/pkg/resource/sdk.go.tpl#L13

It's easier to always use this alias and by using the alias, you allow readers of the code to imagine how this code might be used in other service-specific controllers.